src/components/network/network-map-tab.tsx

```diff
@@ -133,6 +127,7 @@ type NetworkMapTabProps = {
     showInSpreadsheet: (equipment: { equipmentType: EquipmentType; equipmentId: string }) => void;
     onPolygonChanged: (polygoneFeature: any) => void;
     onElementCreated?: () => void;
+    isInDrawingMode: UseStateBooleanReturn;
```

Comment:
Maybe use a boolean type and passe isInDrawingMode.value from parent if you need ?